packages/pigeon/platform_tests/test_plugin/android/src/main/kotlin/com/example/test_plugin/CoreTests.gen.kt

```diff
@@ -270,7 +270,22 @@ interface HostIntegrationCoreApi {
   /** Returns the passed string asynchronously. */
   fun echoAsyncString(aString: String, callback: (String) -> Unit)
   fun callFlutterNoop(callback: () -> Unit)
+  fun callFlutterEchoAllTypes(everything: AllTypes, callback: (AllTypes) -> Unit)
+  fun callFlutterSendMultipleNullableTypes(aNullableBool: Boolean?, aNullableInt: Long?, aNullableString: String?, callback: (AllNullableTypes) -> Unit)
```

Comment:
Do we have a linter/formatter for kotlin? Is this something we can roll in to our tools?

Reply:
We don't have one; filed https://github.com/flutter/flutter/issues/118756